File: ra-back/src/main/java/pl/raptors/ra_back/domain/mapsMetadata/MapMetadata.java
package pl.raptors.ra_back.domain.mapsMetadata;

import lombok.Data;

@Data
public class MapMetadata {      // na wszelki wypadek dane w stringach aby nie powstały żadne zaokrąglenia związane z konwersją
    String image;
    String resolution;
    String[] origin;
    String negate;
    String occupied_thresh;
    String free_thresh;
}

File: ra-front/src/app/model/mapsMetadata/MapMetadata.ts
export class MapMetadata{
    id: string;
    image: string;
    resolution: string;
    origin: string[];
    negate: string;
    occupied_thresh: string;
    free_thresh: string;
    
   constructor(id: string, image: string, resolution: string, origin: string[], negate: string, occupied_thresh: string, free_thresh: string) {
     this.id = id;
     this.image = image;
     this.resolution = resolution;
     this.origin = origin;
     this.negate = negate;
     this.occupied_thresh = occupied_thresh;
     this.free_thresh = free_thresh;
   }
 }
 